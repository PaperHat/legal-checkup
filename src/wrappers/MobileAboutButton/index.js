import React from 'react'
import StyledMobileButton from '@components/styled/MobileButton'

export default function MobileAboutButton () {
  return (
    <StyledMobileButton
      onClick={() => {
        window.open('http://www.wclc.org.nz/about-us/', '_blank')
      }}
    >
      About Community Law
    </StyledMobileButton>
  )
}
