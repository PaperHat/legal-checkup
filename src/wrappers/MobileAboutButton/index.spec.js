import React from 'react'
import { shallow } from 'enzyme'

import MobileAboutButton from '.'

describe('components:wrappers:MobileAboutButton', () => {
  it('renders the MobileAboutButton and CSS properties properly', () => {
    expect(toJson(shallow(<MobileAboutButton />))).toMatchSnapshot()
  })
})
