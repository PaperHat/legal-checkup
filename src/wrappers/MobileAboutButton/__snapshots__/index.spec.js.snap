// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:wrappers:MobileAboutButton renders the MobileAboutButton and CSS properties properly 1`] = `
<StyledMobileButton
  onClick={[Function]}
>
  About Community Law
</StyledMobileButton>
`;
